README.md

```diff
@@ -90,65 +100,122 @@ python3.10 -m pip install -r requirements.txt
-   - Windows: C:\\\\....\\\<username\>\\.irods\\irods_environment.json
-
-- Your iRODS admin will proivide a file `irods_environment.json`. Place that file into the `.irods` folder
-   Example:
-   
-```py
+
+- Please create a directory/folder named `.irods` in your home directory/folder (`~/.irods/` in Linux shorthand).
+  - Linux: `/home/\<username\>/.irods/irods_environment.json`
+  - Mac: `/Users/\<username\>/.irods/irods_environment.json`
+  - Windows: `C:\\\\....\\\<username\>\\.irods\\irods_environment.json`
+
+- Your *iRODS* admin will provide an `irods_environment.json` file, its contents, or instructions on how to create it.  Place that file into the `.irods` directory/folder.  Here it an example that can be created with the `iinit` iCommand on Linux:
+
+```json
 {
-	"irods_host": "server.fqdn.nl", 
-	"irods_port": 1247, 
-	"irods_user_name": "cstaiger", 
-	"irods_zone_name": "npecZone", 
-	"default_resource_name": "disk", 
-	"davrods_server": "https://server.fqdn.nl", 
-	"ui_tabs": [ #Activated tabs for the user
-		"tabBrowser", 
-		"tabUpDownload", #requires "default_resource_name" (not demoResc!)
-		"tabELNData", 
-		"tabDataCompression" #requires "default_resource_name" (not demoResc!)
-	]
+    "irods_host": "server.fqdn.nl", 
+    "irods_port": 1247, 
+    "irods_user_name": "username", 
+    "irods_zone_name": "myZone", 
+    "irods_default_resource": "myResc" 
 }
 ```
 
+### iBridges config.json
 
-## Usage
+*iBridges* will create its own configuration file in `~/.ibridges/` containing the name of the last *iRODS* environment file used.  This `config.json` file can be updated to control other aspects of *iBridges*.  For example:
+
+```json
+{
+    "last_ienv": "irods_environment.json", 
+    "davrods_server": "https://server.fqdn.nl", 
+    "ui_tabs": [ 
+        "tabUpDownload", 
+        "tabELNData", 
+        "tabDataBundle", 
+        "tabCreateTicket" 
+    ], 
+    "force_unknown_free_space": false 
+}
 ```
+Options:
+- `davrods_server`: for annotation of eLabJournal data
+- `ui_tabs`: configure which tabs are shown (Browser and Info tabs always are)
+  - `tabUpDownload`: a two-pane upload/download tab
+  - `tabELNData`: for the Electronic Lab Notebook, eLabJournal
+  - `tabDataBundle`: (un)bundle datasets from/to four supported formats
+  - `tabCreateTicket`: create iRODS tickets for anonymous access
+- `force_unknown_free_space`: ignore if resources' free space is unannotated
+
+The `force_unknown_free_space` option is *REQUIRED* to be set to `true` if your default resource does not yet have its free space annotated.  It makes unannotated top-level resources visible in the drop-downs allowing selection of them.  In addition, it sets the `force` flag for uploads overriding resource overflow protection.
+
+The logs for both GUI and CLI clients can be found in the `~/.ibridges/` directory/folder.
+
+## Usage
+
+```bash
 export PYTHON_IRODSCLIENT_DEFAULT_XML=QUASI_XML
```

Comment:
@chStaiger Is this still necessary?  I added to the environment at the top of `irods-iBridgesGui.py`.  If this works and is necessary, it can be added to `irods-iBridgesCli.py` and removed from here.

Reply:
I will, and after the merge I will have to revise the docs anyways.

Reply:
Just let me know when you have tested enough to merge.